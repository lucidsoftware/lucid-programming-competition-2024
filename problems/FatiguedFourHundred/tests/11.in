version https://git-lfs.github.com/spec/v1
oid sha256:4a27378a291f2ea3a3ecca3b43b3fda192803bcbe9bfcd8958ff3f6ecc824b5c
size 56500
