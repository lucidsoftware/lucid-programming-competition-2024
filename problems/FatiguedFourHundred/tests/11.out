version https://git-lfs.github.com/spec/v1
oid sha256:2d986fd3f3fddbf74ebbf12422f9bb97d80f87b54efe48a3878eab7bad5305c3
size 41626
